src/modules/feature-modules/innovator/services/innovator.service.ts

```diff
@@ -167,9 +167,9 @@ export class InnovatorService extends CoreService {
     );
   }
 
-  pauseInnovation(innovationId: string, message: string): Observable<{ id: string }> {
+  archiveInnovation(innovationId: string, message: string): Observable<{ id: string }> {
     const url = new UrlModel(this.API_INNOVATIONS_URL)
-      .addPath('v1/:innovationId/pause')
+      .addPath('v1/:innovationId/archive')
       .setPathParams({ innovationId });
     return this.http.patch<{ id: string }>(url.buildUrl(), { message }).pipe(
```

Comment:
Now this method doesn't return the id, now is a `204`. You can change the return type to `void`.

Reply:
You indeed had already told me this. Thanks for the reminder!